src/compat.jl

```diff
@@ -22,6 +22,7 @@ __init__() = supported() || compat_warn()
     const compat_contains = Base.contains
```

Comment:
need to update the `v07 = v"0.7.0-DEV.4366"` just above here?

Reply:
done